lib/rouge/lexers/hocon.rb

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*- #
+# frozen_string_literal: true
+
+module Rouge
+  module Lexers
+    load_lexer 'json.rb'
+
+    class HOCON < JSON
+      title 'HOCON'
+      desc "Human-Optimized Object Configuration Notation (https://github.com/lightbend/config)"
+      tag 'hocon'
+      filenames '*.hocon'
+
+      prepend :root do
+        # Comments
+        rule %r(//.*?$), Comment::Single
+        rule %r(#.*?$), Comment::Single
+
+        # Interpolation
+        rule %r/[$][{][?]?/, Literal::String::Interpol, :interpolation
+
+        # Strings
+        rule %r/"""/, Literal::String::Double, :multiline_string
+
+        # Keywords
+        rule %r/\b(?:include|url|file|classpath)\b/, Keyword
+
+        # Symbols
+        rule %r/[()={},:\[\]]/, Punctuation
+
+        # Keys
+        rule %r/([\w\-\.]+? *)([{:=]|\+=)/ do
+          groups Name::Attribute, Punctuation::Indicator
+        end
+
+        # Numbers (handle the case where we have multiple periods, ie. IP addresses)
+        rule %r/\d+\.(\d+\.?){3,}/, Literal #
+
+        # Values
+        rule %r/[^\$\"{}\[\]:=,\+#`\^\?!@\*&]+?/, Literal
```

Comment:
These values aren't picked up by the JSON lexer?

Reply:
Without this rule, unquoted strings aren't highlighted correctly, like this line:

```
retries_msg = You have ${databases.mysql.retries} retries
```

Reply:
Makes sense.